app/views/facility_reservations/timeline.html.haml

```diff
@@ -16,7 +16,7 @@
       = link_to "&laquo;".html_safe,
         timeline_facility_reservations_path(current_facility, date: format_usa_date(@display_date - 1.day)), id: "reservation_left"
       #reservation_date
-        = human_date_with_day(@display_date)
+        = l(@display_date.to_date, format: :timeline_navigation)
```

Comment:
Just curious, would it be possible to pull the `#to_date` a little further up so `@display_date` is always already a date?